bibliopixel/gamma.py

```diff
@@ -4,7 +4,10 @@ def __init__(self, gamma=1.0, offset=0, lower_bound=0):
         self.offset = offset
         self.lower_bound = lower_bound
         width = 255 - lower_bound
-        gam = lambda i: int(lower_bound + pow(i / 255, gamma) * width + offset)
```

Comment:
I sort of agree that the new version is better, but does flake8 forbid lambdas?!